index.html

```diff
@@ -5510,14 +5524,65 @@ <h2 id="instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
             <li>Set <a>processor reference</a> in <var>this</var> to
             <var>processorInstance</var>.
             </li>
-            <li>Add <var>nodeInstance</var> to the <a>node instance set</a> in
-            the <a>BaseAudioContext</a>.
-            </li>
             <li>Return <var>this</var> as a result of the construction.
             </li>
           </ol>
         </section>
         <section class="informative">
+          <h2 id="AudioWorklet-Sequence">
+            AudioWorklet Sequence of Events
+          </h2>
+          <p>
+            The following figure illustrates an idealized sequence of events
+            occurring relative to an <a>AudioWorklet</a>:
+          </p>
+          <figure>
+            <img alt="AudioWorklet sequence" src=
+            "images/audioProcessorInstantiation.png" width="917" height="546">
+            <figcaption>
+              <a><code>AudioWorklet</code></a> sequence
+            </figcaption>
+          </figure>
+          <p>
+            The steps depicted in the diagram are one possible sequence of
+            events involving the creation of an <a>AudioContext</a> and an
+            associated <a>AudioWorkletGlobalScope</a>, followed by the creation
+            of an <a>AudioWorkletNode</a> and its associated
+            <a>AudioWorkletProcessor</a>.
+          </p>
+          <ol>
+            <li>In the main scope, <code>window.audioWorklet</code> is
+            requested to import a script. No <a>AudioWorkletGlobalScope</a>s
+            exist yet, so the script is fetched and added to the Worklet module
+            responses map.
+            </li>
+            <li>An <a>AudioContext</a> is created.
+            </li>
+            <li>An <a>AudioWorkletGlobalScope</a> is created in association
+            with the context's audio rendering thread.
+            </li>
+            <li>As part of the global scope's initialization, the set of
+            imported module scripts is run, including the one that was
+            previously imported.
+            </li>
+            <li>As part of running the imported script, an
+            <a>AudioWorkletProcessor</a> is registered under the key
+            <code>"Custom1"</code> within the <a>AudioWorkletGlobalScope</a>.
+            </li>
+            <li>In the main scope, an <a>AudioWorkletNode</a> is created using
+            the key <code>"Custom1"</code> along with an <code>opts</code>
+            dictionary of options.
+            </li>
+            <li>As part of the node's creation, this key is used to look up the
+            correct <a>AudioWorkletProcessor</a> subclass for instantiation.
+            </li>
+            <li>An instance of the <a>AudioWorkletProcessor</a> subclass is
+            instantiated with a structured clone of the same <code>opts</code>
+            dictionary.
```

Comment:
When I reached here, this section reads like a rough description of 'instantiation process'. Perhaps we should focus on explaining relationship between classes.